Add ga (#25)
arcdocs/aire: book/privacy_and_cookies.md

## 2. Changes to this Notice

We keep the information provided in this Notice under review. This Notice may be updated from time to time. If we make any substantial updates, we will draw these to your attention.
We will not use your personal data in any manner that is incompatible with the purpose for which the data were collected originally, unless we have obtained your consent to that additional use.

## 3. Anything you are not clear about

If there is anything you are unclear about, please contact our Data Protection Officer, who shall be happy to answer any queries you may have concerning this Notice or the way in which we process your personal data.
The Data Protection Officer's contact details are provided at the end of this Notice.

## 4. Purpose of processing your data

Where we request personal data from you on our website you will be informed at that time of the purpose for which the data will be processed and how it will be processed, its legal basis for processing as well as any transfers to third parties.

## 5. Purpose of processing - automated collection of personal information

> When you access our web pages certain information your browser provides, including your IP address at the time, browser type, and potentially the address of the page you last visited, will be automatically recorded by the University [if externally hosted add details here of any collection carried out by the hosting provider]. This data may be used to aid detection in the event of a security breach. The legal basis for this processing is necessity to pursue our legitimate interests to keep our website secure. The University [if externally hosted add details here of any collection carried out by the hosting provider] retains this data in a form from which you may be identifiable for a period of up to one month.

Reviewer: Shall we get rid of these bracketed bits?

Author: ```suggestion
When you access our web pages certain information your browser provides, including your IP address at the time, browser type, and potentially the address of the page you last visited, will be automatically recorded by the University and GitHub. This data may be used to aid detection in the event of a security breach. The legal basis for this processing is necessity to pursue our legitimate interests to keep our website secure. The University and GitHub retains this data in a form from which you may be identifiable for a period of up to one month.
```
